h2o-core/src/main/java/water/parser/SVMLightFVecParseWriter.java

```diff
@@ -37,11 +37,13 @@ public void newLine() {
   @Override public void addStrCol(int idx, BufferedString str){addInvalidCol(idx);}
   @Override public boolean isString(int idx){return false;}
   @Override public FVecParseWriter close(Futures fs) {
+    if (_nvs != null) {
```

Comment:
Why is this change here? The version in master looks ok.
(if( _nvs == null ) return this; // Might call close twice)